composer.json

```diff
@@ -26,7 +26,8 @@
     "require": {
         "php": "~5.4",
         "doctrine/mongodb-odm": "~1.0@beta",
-        "xiag/rql-parser": "^1.0"
+        "xiag/rql-parser": "^1.0",
+        "symfony/event-dispatcher": "^2.6"
```

Comment:
@hairmare I'm a bit concerned about the deps.. Currently, this pulled `symfony/event-dispatcher-v2.7.2`. In graviton, we require symfony et all with `"symfony/symfony": "~2.6.5"`; so 2.7.2 of anything `symfony/*` cannot enter there as they would not be required of each other.

Don't we run into a risk of using a `EventDispatcher` API that is 2.7.* specific while this could not be satisfied in context of the graviton deps?